premailer/premailer.py

```diff
@@ -510,6 +514,24 @@ def _load_external(self, url):
 
         return css_body
 
+    @staticmethod
+    def six_colour(colourValue):
+        '''Fix background colours for Lotus Notes
+
+        Notes which fails to handle three character ``bgcolor`` codes well.
+        see <https://github.com/peterbe/premailer/issues/114>'''
+
+        shortCodes = re.compile(r'^#([0-9A-F])([0-9A-F])([0-9A-F])$', re.I)
+        # double digits to enlongen color code
+        retval = shortCodes.sub(r'#\1\1\2\2\3\3', colourValue)
+
+        # Drop "transparent" bgcolor values entirely
+        # The space at the start of the RE is deliberate
+        transparentColor = re.compile(r'transparent', re.I)
+        if transparentColor.match(retval):
+            raise TransparentIsNotAColour(retval)
+        return retval
```

Comment:
Would you mind pep8 fix this? E.g instead of "shortCodes", make it "short_codes"